src/app/components/file-browser/file-list/file-list.component.ts

```diff
@@ -13,27 +13,29 @@ import {
     ViewChild
 } from '@angular/core';
 import { Subscription } from 'rxjs/Subscription';
-import { TerraStorageObjectList } from '../model/terra-storage-object-list';
-import { TerraSimpleTableHeaderCellInterface } from '../../tables/simple/cell/terra-simple-table-header-cell.interface';
-import { TerraSimpleTableRowInterface } from '../../tables/simple/row/terra-simple-table-row.interface';
-import { TerraStorageObject } from '../model/terra-storage-object';
 import * as moment from 'moment';
 import { TerraBaseStorageService } from '../terra-base-storage.interface';
-import { TerraButtonInterface } from '../../buttons/button/data/terra-button.interface';
-import { PathHelper } from '../helper/path.helper';
 import { TerraFileBrowserComponent } from '../terra-file-browser.component';
-import { FileType } from '../helper/fileType.helper';
-import { TerraSimpleTableComponent } from '../../tables/simple/terra-simple-table.component';
 import { TerraFileBrowserService } from '../terra-file-browser.service';
-import { TerraUploadItem } from '../model/terra-upload-item';
-import { ClipboardHelper } from '../helper/clipboard.helper';
-import { TerraSimpleTableCellInterface } from '../../tables/simple/cell/terra-simple-table-cell.interface';
 import { TranslationService } from 'angular-l10n';
 import {
     isNull,
     isNullOrUndefined
 } from 'util';
 import { TerraBasePrivateStorageService } from '../terra-base-private-storage.interface';
+import {
+    TerraButtonInterface,
+    TerraSimpleTableCellInterface,
+    TerraSimpleTableComponent,
+    TerraSimpleTableHeaderCellInterface,
+    TerraSimpleTableRowInterface,
+    TerraStorageObject,
+    TerraStorageObjectList,
+    TerraUploadItem
+} from '../../../../';
+import { FileType } from '../../../helpers/fileType.helper';
```

Comment:
Ist eine Umbenennung des Helpers in `FileTypeHelper` vllt sinnvoll? Alle anderen Helper haben diesen Suffix

Reply:
Guter Einwand 👍